language-configuration.json

```diff
@@ -15,8 +15,9 @@
     ],
     // symbols that are auto closed when typing
     "autoClosingPairs": [
-        ["{{", "}}"],
```

Comment:
@ahmadalli I think you just need to remove the double bracket pairs because the single bracket will handle that.

Reply:
yeah that's doable too